helm/csi-vxflexos/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "2.1.0"
-kubeVersion: ">= 1.20.0 < 1.23.0"
+kubeVersion: ">= 1.20.0 < 1.24.0"
```

Comment:
I think we should change the min version supported to be 1.21 as well. We only support 2 revs back if i am not mistaken and all the qual plan we have states 1.21 as min.

Reply:
Fixed